.travis.yml

```diff
@@ -1,43 +1,43 @@
     - docker login -u $DOCKER_USERNAME -p $DOCKER_CODE
     - docker image push virendras19/todo-app:test
-
   - stage: Test
     before_script: docker pull virendras19/todo-app:test
-    script: docker run --mount type=bind,source="$(pwd)"/,target=/app/ virendras19/todo-app:test tests/test_view_model.py
-    name: "Unit Test"
-
-   
-  - before_script: docker pull virendras19/todo-app:test 
-    script: docker run --mount type=bind,source="$(pwd)"/,target=/app/ virendras19/todo-app:test tests/test_endpoints.py
-    name: "Endpoint Test"
-    
-  - before_script: docker pull virendras19/todo-app:test 
-    script: docker run  -eT_KEY=$T_KEY -eT_TOKEN=$T_TOKEN --mount type=bind,source="$(pwd)"/,target=/app/ virendras19/todo-app:test tests_e2e
-    name: "End to End Tests"
-
+    script: docker run --mount type=bind,source="$(pwd)"/,target=/app/ virendras19/todo-app:test
+      tests/test_view_model.py
+    name: Unit Test
+  - before_script: docker pull virendras19/todo-app:test
+    script: docker run --mount type=bind,source="$(pwd)"/,target=/app/ virendras19/todo-app:test
+      tests/test_endpoints.py
+    name: Endpoint Test
+  - before_script: docker pull virendras19/todo-app:test
+    script: docker run  -eM_KEY=$M_KEY -eM_TOKEN=$M_TOKEN --mount type=bind,source="$(pwd)"/,target=/app/
+      virendras19/todo-app:test tests_e2e
+    name: End to End Tests
   - stage: Deploy
     before_script:
     - docker login -u $DOCKER_USERNAME -p $DOCKER_CODE
-    - docker login --username=virendrasankpal@yahoo.com --password=$(heroku auth:token) registry.heroku.com
+    - docker login --username=virendrasankpal@yahoo.com --password=$(heroku auth:token)
+      registry.heroku.com
     - heroku container:login
     script:
-    - docker build --target production --tag virendras19/todo-app:latest --tag virendras19/todo_app:$TRAVIS_COMMIT .
+    - docker build --target production --tag virendras19/todo-app:latest --tag virendras19/todo_app:$TRAVIS_COMMIT
+      .
     - docker image push virendras19/todo-app:latest
     - docker tag virendras19/todo-app registry.heroku.com/virendra-todo-app/web
     - docker push registry.heroku.com/virendra-todo-app/web
     - heroku container:release web --app virendra-todo-app
-
     on:
       branch: module-8
- 
 env:
   global:
   - secure: j0qzXTqWRDev723kRcNFRu5WVrr/9I/inGGvGF7oq6gRtwplTgORF+xGq5FFeOwXcg1+18U5mDz+DbJ9unBVG4F+Kvg29pIHkZ6rl+SYzfijyc0mxV/2U3YKQFFgNRDDtcl/rGXZNsi2gLvT0QTOIPmZecn2rWziIDLte/XupcdG6P2xdlzAIV8BVtqDzq3I3VTXkpYCAW9l3Vgjmc0yguNsCf2RhwRRkPHRwDHtgz6R2MWEJ1AYWuPPFRHf6veEyAQphk2t45MTFjITkcaNqEiqelnIfgdV0qcnpjCLGoz/o8g+eK88KuAph1SQJq0dTFEHkVMixMLXGWag2/B0e3mSvwl0D792libtZUIqqxanciaXXqTors6r6LZuZIIxixO8d95JUaFmsnwBGLvRcg6NQYe6FhJx/MRIR4IBoOkDBBoRdR9kCqNeWNF+nRv4hvJ1e+0UsFe38R5N1TVfgMkNfak6ZXypmwt216xwgU81xnLyNOj2LqxeXKWQiZVvZ5ggDWjCTQa9o7bY1u2V9EHPjaAH3z4hyyx6babP1o4VedLvSqKvHQ4A2DDN5dIjlc8WYVEkjwZGrmEGAJ84nt7OWkASk5aeHFPYhtSIwjWF/KBIkNaVEzhIKmLhJ8qO1a8MGtFVJMwd6t7yoYHwULDz382Id7l8vJB1L2z6A5Q=
```

Comment:
It would be good to now remove the Trello credentials that you have encrypted here!